From 908c388c922c6369cace0b76660198becee2284e Mon Sep 17 00:00:00 2001
From: Kamil Aronowski <kamil.aronowski@yahoo.com>
Date: Mon, 8 May 2023 10:15:21 +0200
Subject: [PATCH 18/35] Change type of fallback_verbose_wait from int to
 unsigned long

The variable fallback_verbose_wait from now on is of the type unsigned
long to match the type of the argument usleep() accepts.

Signed-off-by: Kamil Aronowski <kamil.aronowski@yahoo.com>
---
 fallback.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fallback.c b/fallback.c
index ad70ccf..600cc7a 100644
--- a/fallback.c
+++ b/fallback.c
@@ -1006,7 +1006,7 @@ try_start_first_option(EFI_HANDLE parent_image_handle)
 	EFI_HANDLE image_handle;
 
 	if (get_fallback_verbose()) {
-		int fallback_verbose_wait = 500000; /* default to 0.5s */
+		unsigned long fallback_verbose_wait = 500000; /* default to 0.5s */
 #ifdef FALLBACK_VERBOSE_WAIT
 		fallback_verbose_wait = FALLBACK_VERBOSE_WAIT;
 #endif
@@ -1211,7 +1211,7 @@ reset:
 	console_print(L"Reset System\n");
 
 	if (get_fallback_verbose()) {
-		int fallback_verbose_wait = 500000; /* default to 0.5s */
+		unsigned long fallback_verbose_wait = 500000; /* default to 0.5s */
 #ifdef FALLBACK_VERBOSE_WAIT
 		fallback_verbose_wait = FALLBACK_VERBOSE_WAIT;
 #endif
-- 
2.39.0

